test/lhash_test.c

```diff
@@ -15,6 +15,7 @@
 #include <openssl/lhash.h>
 #include <openssl/err.h>
 #include <openssl/crypto.h>
+#include <internal/hashtable.h>
```

Comment:
We should have a fuzzer too IMO.

Reply:
added a fuzzer, will build the corpus and submit it separately in a bit